public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
+  "avatar": "https://github.com/ShlokMohanty.png",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "http://github.com/ShlokMohanty",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohanty_shlok",
+      "icon": "twitter"
+    },
+   {
+    "name": "Follow me on Facebook",
+    "url": "https://www.facebook.com/shlok.mohanty.35",
+    "icon": "facebook"
+    }
+  ]
+}
+  
```

Comment:
```suggestion
```

Reply:
@Panquesito7 do i have to make any changes or changes have already been made?

Reply:
Resolve this conversation